packages/amplify-category-function/src/provider-utils/awscloudformation/secrets/functionSecretsStateManager.ts

```diff
@@ -244,10 +257,37 @@ const setLocalFunctionSecretState = (functionName: string, secretDeltas: SecretD
     secretNames: existingSecrets,
   };
   const parametersFilePath = path.join(pathManager.getBackendDirPath(), categoryName, functionName, functionParametersFileName);
+
   // checking for existance of the file because in the case of function deletion we don't want to create the file again
   if (fs.existsSync(parametersFilePath)) {
+    const params = JSONUtilities.readJson<{ secretNames?: string[] }>(parametersFilePath, { throwIfNotExist: false }) || {};
```

Comment:
I don't see params used, but if it is needed, it would be nice if this was a stateManager call instead

Reply:
ah yeah this can be removed. leftover from some experimenting